packages/lesswrong/lib/collections/comments/helpers.ts

```diff
@@ -61,8 +57,8 @@ export function commentGetPageUrlFromIds({postId, postSlug, tagSlug, tagCommentT
     } else {
       return `${prefix}/posts/${postId}/${postSlug?postSlug:""}#${commentId}`;
     }
-  } else if (tagSlug) {
-    return tagGetCommentLink(tagSlug, commentId, tagCommentType, isAbsolute);
+  } else if (tagSlug && tagCommentType) {
+    return tagGetCommentLink({tagSlug, commentId, tagCommentType, isAbsolute});
```

Comment:
Maybe we should default to `DISCUSSION` type rather than checking for it to exist?